pydy/codegen/ode_function_generators.py

```diff
@@ -176,7 +176,7 @@ def _deduce_system_type(**kwargs):
 
         return system_type
 
-    def __init__(self, right_hand_side, coordinates, speeds, constants,
+    def __init__(self, right_hand_side, coordinates, speeds, constants=[],
```

Comment:
This should be `constants=()`. See http://docs.python-guide.org/en/latest/writing/gotchas/